### DISCUSSION TITLE
FR: multisite provide siteId for entries

### DISCUSSION THREAD

**Discussion by RichardJong** [engram-design/FeedMe#878] [Ideas]
For a multisite I'd like to provide a siteId for the entries I'm importing.

E.g.:

```
{
    "entry": [
        {
            "Title": "Guide first entry example",
            "siteId":"site1"
        },
        {
            "Title": "Guide second entry example",
            "siteId":"site2"
        },
    ]
}
```

**Comment by engram-design** [comment]
For now, you'll need to split the entries to import into their site, and create two different feeds. We'll add it to the feature list.

**Comment by codyjames** [comment]
Would love this as well.

**Comment by naboo** [comment]
Would like to +1 this old gem! :)
